src/TypeScript.UnitTests/EslintBridgeClient/AnalysisConfigurationTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Linq;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.UnitTests;
+using SonarLint.VisualStudio.TypeScript.EslintBridgeClient;
+
+namespace SonarLint.VisualStudio.TypeScript.UnitTests.EslintBridgeClient
+{
+    [TestClass]
+    public class AnalysisConfigurationTests
+    {
+        [TestMethod]
+        public void GetGlobals_NoEnvVar_ReturnsDefaults()
+        {
+            var testSubject = CreateTestSubject();
```

Comment:
This test assumes the environment variable is not set, which might not be the case when running on a dev machine.

Any tests that use environment variables directly should use an `EnvironmentVariableScope`.